jboss-kie-kieserver/tests/bats/jboss-kie-kieserver-hooks.bats

```diff
@@ -32,6 +32,7 @@ teardown() {
 export RHPAM_CENTRAL_CONSOLE_SERVICE_HOST="localhost"
 export RHPAM_CENTRAL_CONSOLE_SERVICE_PORT_HTTP="8080"
 export WORKBENCH_SERVICE_NAME="rhpam-central-console"
+export KIE_SERVER_STARTUP_STRATEGY=OpenShiftStartupStrategy
```

Comment:
```suggestion
export KIE_SERVER_STARTUP_STRATEGY="OpenShiftStartupStrategy"
```

Reply:
Updated as per suggested

Reply:
Wasn't really necessary, but doesn't hurt. Outcome is the same.